person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services?
+    @age >= 18 or @parent_permissiond == true
+  end
```

Comment:
#### [OPTIONAL] because the previous reviewer didn't notice this the following suggestions are NOT mandatory. I do recommend strongly for you to have a look at it.
- [ ] according to the requirements, method `of_age?()` should be private.
- [ ] method `can_use_services?` on the other hand should be Public and it should check if the person is of age using the private method `of_age?()` or if they have permission from the parents.
- [ ] lastly according to the Ruby requirements The `and` and `or` keywords are banned. It's just not worth it. Always use `&&` and `||` instead.

Reply:
Thank you, @mikethreels, for reviewing my pull request. I want to ask you about the private method. I defined `of_age` method and I made it private by using `private :of_age?`. Is it private or should I use another approach to make it private?

Reply:
No problem at all you can just put the keyword `private`, all the methods you put below that keyword will be private something like in the below example:
```
  def can_use_services?
    of_age?() || @parent_permissiond
  end

  private

  def of_age?
    @age >= 18
  end
```